static/js/lib/form.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import type { CheckoutPayload } from "../flow/ecommerceTypes"
+
+/**
+ * Creates a POST form with hidden input fields
+ * @param url the url for the form action
+ * @param payload Each key value pair will become an input field
+ */
+export function createForm(
```

Comment:
Rename to something more specific like `createCheckoutForm` or `createCybersourceForm`?

Reply:
:+1: